src/components/modals/HarvestAll.tsx

```diff
@@ -131,6 +134,36 @@ export const HarvestAllModal: FC<IHarvestAllModalProps> = ({
 		});
 	}, [network]);
 
+	useEffect(() => {
+		console.log('==================================');
+		console.log('givDropClaimed', currentBalance.givDropClaimed);
+		console.log(
+			'network === config.XDAI_NETWORK_NUMBER ',
+			network === config.XDAI_NETWORK_NUMBER,
+		);
+		console.log(
+			'condition',
+			network === config.XDAI_NETWORK_NUMBER &&
+				!currentBalance.givDropClaimed,
+		);
+
+		if (
+			network === config.XDAI_NETWORK_NUMBER &&
+			!currentBalance.givDropClaimed
+		) {
+			fetchAirDropClaimData(address).then(claimData => {
+				if (claimData) {
+					console.log(`claimData`, claimData);
```

Comment:
Remove this line and other console.log lines please

Reply:
they removed in last commit.